lib/reveal-ck/config.rb

```diff
@@ -50,7 +50,8 @@ def filter_defaults
                       'HTML::Pipeline::MentionFilter',
                       'HTML::Pipeline::AutolinkFilter'],
         'asset_root' => 'https://assets-cdn.github.com/images/icons/',
-        'base_url' => 'https://github.com'
+        'base_url' => 'https://github.com',
+        'requires' => []
```

Comment:
I haven't played around with this yet, but I noticed this entry is `requires` (plural) but up on line 39 of create_slides_html.rb in this PR it's `config.require` (singular). Is that on purpose?